en/docs/learn/configuring-roles-and-permissions-for-a-service-provider.md

```diff
@@ -63,6 +63,12 @@ Next, follow the steps below to configure roles and define custom permission for
         When you add role mapping, you can map an exact role name available
         in WSO2 IS to a particular service provider role.
     
+4. Optionally, you can also enable the following configuration in the `<IS_HOME>/repository/conf/deployment.toml` file to return only roles that you have mapped under this service provider. This allows responses such as ID tokens or SAML assertions to return only the mapped SP role of the user instead of returning all user roles. 
+
```

Comment:
... to return only the list of mapped SP roles from the assigned roles instead of returning all user roles.

Reply:
fixed with 1c354532a37132deb4767c650a9f969bac5cf4a5